requirements.txt

```diff
@@ -7,12 +7,12 @@ scikit-build
 scipy
 dicompyler-core
 pydicom
-pyradiomics
 PyQt5
 country-list
 git+https://github.com/matplotlib/legacycontour.git
 fuzzywuzzy
 matplotlib==3.2.0
 PyWavelets==0.5.2
 dataclasses==0.7;python_version=="3.6.*"
-networkx
\ No newline at end of file
+networkx
+pytest-qt

```

Comment:
Is this something every user needs to have, or just developers.
It isn't unusual to have testing related packages dealt with separately (in a developer oriented requirements-dev.txt or something like that).